packages/date-picker/src/vaadin-date-picker-overlay-content-mixin.js

```diff
@@ -182,9 +174,6 @@ export const DatePickerOverlayContentMixin = (superClass) =>
 
     /** @protected */
     _addListeners() {
-      setTouchAction(this.$.scrollers, 'pan-y');
```

Comment:
This was needed for scrolling to work while using `track` gesture, according to [Polymer docs](https://polymer-library.polymer-project.org/2.0/docs/devguide/gesture-events#gestures-and-scroll-direction):

> Listening for certain gestures controls the scrolling direction for touch input. For example, nodes with a listener for the track event will prevent scrolling by default.

> If you want use Polymer gesture events and native gesture handling, you can use the Polymer.Gestures.setTouchAction function to specify which events the browser should handle natively. For example, if you want the browser to handle vertical scrolling, but have your element handle left-right swiping actions, you could do something like this